packages/component-boilerplate/test/main.ts

```diff
@@ -0,0 +1,473 @@
+import { setImplementation } from "@saasquatch/universal-hooks";
+// use React testing lib until i implement async utils
+// which may or may not happen
+// import {
+//   act,
+//   renderHook,
+//   setTestImplementation,
+// } from "@saasquatch/universal-hooks-testing-library";
+import * as React from "react";
+// import * as ReactTestLib from "@testing-library/react-hooks";
+import { act, renderHook } from "@testing-library/react-hooks";
```

Comment:
Swap to the universal hooks testing library, and test across both Haunted and React?

Reply:
haunted doesn't run nicely with jest at the moment, we can sort that out another time